sklearn/decomposition/_dict_learning.py

```diff
@@ -113,29 +110,9 @@ def _sparse_encode(
     -------
     code : ndarray of shape (n_components, n_features)
         The sparse codes.
-
-    See Also
-    --------
-    sklearn.linear_model.lars_path
-    sklearn.linear_model.orthogonal_mp
-    sklearn.linear_model.Lasso
-    SparseCoder
     """
-    if X.ndim == 1:
-        X = X[:, np.newaxis]
```

Comment:
It looks like this line has not been moved. Yet, do we still need this?

Reply:
This was in `_sparse_encode` which was only called by ``sparse_encode`` which already took care of providing 2D arrays so it was just dead code.